### PR TITLE
Feat/tracking

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | MANAGER-5135
| License          | BSD 3-Clause

## Description

Use common configuration for tracking and update level2

### CODE REVIEW

**Comment by cbourgois** [comment]
Some conflicts in `package.json`s concerning `@ovh-ux/manager-account-sidebar`

**Comment by marie-j** [comment]
@cbourgois Thanks for pointing it out, I've rebased to solve conflicts

File: packages/manager/apps/cloud/client/components/at-internet/at-internet.constants.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
-angular.module('managerApp').constant('TRACKING', {
+export const TRACKING = {
   EU: {
     config: {
-      level2: '10', // 1 is Cloud project ID in AT-Internet EU manager
+      level2: '85', // 1 is Cloud project ID in AT-Internet EU manager
```
Comment (FredericEspiau) [inline packages/manager/apps/cloud/client/components/at-internet/at-internet.constants.js:4]: Maybe you could update the comment as well 😅

**Comment by cbourgois** [comment]
@marie-j, It's Groundhog Day! 🎉  

Some conflicts in `@ovh-ux/manager-public-cloud` `package.json` concerning `@ovh-ux/manager-cloud-styles`

File: packages/components/ng-at-internet/src/constants.js [hunk 2]
```diff
@@ -33,6 +33,11 @@ export const AT_INTERNET_CUSTOM_VARS = {
     path: 'site.6', // OVH's AtInternet configuration
     format: '[%s]',
   },
+
+  siteName: {
+    path: 'site.13', // OVH's AtInternet configuration
```
Comment (antleblanc) [inline packages/components/ng-at-internet/src/constants.js:38]: Not yours but I think we can take this opportunity to rename `OVH` into `OVHcloud` 😎.

File: packages/manager/apps/cloud/client/components/at-internet/at-internet.config.js [hunk 3]
```diff
@@ -1,32 +1,17 @@
...
-    atInternetProvider.setEnabled(trackingEnabled);
-    atInternetProvider.setDebug(!trackingEnabled);
-
-    atInternetUiRouterPluginProvider.setTrackStateChange(true);
-    atInternetUiRouterPluginProvider.addStateNameFilter((routeName) => {
-      const prefix = 'cloud';
-      const route = routeName ? routeName.replace(/\./g, '::') : '';
-      return `${prefix}::${route}`;
-    });
+  .config((atInternetConfigurationProvider) => {
```
Comment (antleblanc) [inline packages/manager/apps/cloud/client/components/at-internet/at-internet.config.js:5]: We can take the opportunity to add some missing ngInject I think.

```suggestion
  .config(/* @ngInject */ (atInternetConfigurationProvider) => {
```

File: packages/manager/modules/at-internet-configuration/.eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "globals": {
+    "__NODE_ENV__": true
```
Comment (antleblanc) [inline packages/manager/modules/at-internet-configuration/.eslintrc.json:3]: Might be better:)
```suggestion
    "__NODE_ENV__": "readonly"
```

File: packages/manager/modules/at-internet-configuration/src/provider.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  setReplacementRules(rules) {
+    this.stateRules = rules;
+  }
+
+  setPrefix(prefix) {
+    this.prefix = prefix;
+  }
+
+  $get() {
```
Comment (antleblanc) [inline packages/manager/modules/at-internet-configuration/src/provider.js:20]: Missing annotation I think.

```suggestion
    /* @ngInject */
    $get() {
```

**Comment by antleblanc** [comment]
@marie-j I didn't notice during my first review but there is a conflict. Could you please resolve it?

Thank you!

File: packages/manager/modules/at-internet-configuration/src/provider.js (hunk 5 above)
Comment (marie-j) [reply packages/manager/modules/at-internet-configuration/src/provider.js:20]: Nothing is passed as parameter so nothing needs to be injected here

**Comment by marie-j** [comment]
Rebased to fix conflicts

**Comment by marie-j** [comment]
Rebased to fix conflicts

**Comment by ovh-cds** [comment]
CDS Report installation#13870.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report build#13995.0 ✘  
* 
  * Build ✘

**Comment by marie-j** [comment]
Rebase to fix conflicts

**Comment by marie-j** [comment]
Rebase to fix conflicts

**Review by cbourgois — DISMISSED** [review]

**Review by cbourgois — APPROVED** [review]

**Pull request merged**